### PR TITLE
[16.0] [OU-ADD] crm

### PR DESCRIPTION
**Pull request by marielejeune**

### CODE REVIEW

**Comment by marielejeune** [comment]
https://github.com/OCA/OpenUpgrade/issues/3681

**Comment by pedrobaeza** [comment]
/ocabot migration crm

<dependency></dependency>

**Review by remytms — APPROVED** [review]
LGTM

File: openupgrade_scripts/scripts/crm/16.0.1.8/pre-migration.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+              AND parent.team_id IS NOT NULL
+        ) as res(partner_id, parent_team_id)
+        WHERE p.id = res.partner_id
+        """,
+    )
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, _column_renames)
```
Comment (daiduongnguyen-odoo) [inline openupgrade_scripts/scripts/crm/16.0.1.8/pre-migration.py]: i think It should be 
```suggestion
    openupgrade.rename_fields(env, _column_renames)
```
to support translation also although the `openupgrade lib` not support yet

Comment (marielejeune) [reply openupgrade_scripts/scripts/crm/16.0.1.8/pre-migration.py]: Right, done

**Review by daiduongnguyen-odoo — APPROVED** [review]

**Pull request merged**